src/vmm/src/devices/virtio/iovec.rs

```diff
@@ -124,8 +125,8 @@ impl IoVecBuffer {
             }
 
             let mut slice =
-            // SAFETY: the constructor IoVecBufferMut::from_descriptor_chain ensures that
-            // all iovecs contained point towards valid ranges of guest memory
+                // SAFETY: the constructor IoVecBufferMut::from_descriptor_chain ensures that
```

Comment:
nit: Would you mind fixing this indentation issue in the commit that introduces the comment?

Reply:
fixed